.github/scripts/manage_invocation.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+import argparse, sys, logging
+from distantrs import Invocation
+
+INVOCATION_DETAILS = '/tmp/distant-rs-invocation.txt'
+
+logging.basicConfig(format="[%(asctime)s] %(levelname)-8s| %(message)s")
+
+l = logging.getLogger(__name__)
+
+def open_i_func(arg):
+    i = Invocation()
+    i.open()
+
+    l.info(f'ID: {i.invocation_id}')
+    l.info(f'Token: {i.auth_token}')
+
+    with open(INVOCATION_DETAILS, 'w') as f:
+        f.write(f'{i.invocation_id}--{i.auth_token}')
+
+def close_i_func(arg):
+    i_details = []
+    ret = 5 if arg.return_code == 0 else 6
+    with open(INVOCATION_DETAILS, 'r') as f:
```

Comment:
some `try/except` in file handling? just to log something sensible

Reply:
Done.